### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by lenny-g**
project updated

### CODE REVIEW

File: src/controllers/api/notes.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  const note = notes.find((note) => note.id === id);
+
+  if (!note) {
+    return res.status(404).json({
+      message: `No note with id: ${id}`,
+    });
+  }
+
+  const newNotes = notes.filter((note) => {
+    return note.id !== id;
```
Comment (Nazehs) [inline src/controllers/api/notes.js:51]: Pls refactor this by removing the return keyword and the curly bracket

File: src/db/db.json [hunk 2]
```diff
@@ -0,0 +1 @@
+[]
```
Comment (LeeFarnell2U) [inline src/db/db.json:1]: This is what is throwing the errors when you first load the page, as there is nothing here to load. Consider putting in  a pre-filled note or something to remove those errors.

File: src/controllers/api/notes.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
+const { v4: uuidv4 } = require("uuid");
+const { getNotesFromFile, writeNotesToFile } = require("../../utils.js");
+
+const getNotes = (req, res) => {
+  const notes = getNotesFromFile();
+  console.log(notes);
```
Comment (LeeFarnell2U) [inline src/controllers/api/notes.js:6]: remove this

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+app.use(express.static("public"));
+
+app.use(express.urlencoded({ extended: true }));
+
+app.use(express.json({ extended: true }));
+
+app.use(routes);
+
+app.listen(PORT, () =>
+  console.log(`server running on https://localhost:${PORT}`)
```
Comment (LeeFarnell2U) [inline src/index.js:19]: This is good practice for when you're a dev, but please remove from the live application

File: src/utils.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+  const notes = JSON.parse(notesJSON);
+  return notes;
+};
+
+const writeNotesToFile = (data) => {
+  try {
+    fs.writeFileSync(path.join(__dirname, "/db/db.json"), data);
+  } catch (error) {
+    console.log(error.message);
```
Comment (LeeFarnell2U) [inline src/utils.js:19]: Rather than a console.log, try using a cosole.error here instead.

**Pull request merged**